flinkdatasource/src/main/java/org/zuinnote/flink/bitcoin/BitcoinTransactionFlinkInputFormat.java

```diff
@@ -35,55 +35,51 @@
 
 
 public class BitcoinTransactionFlinkInputFormat extends AbstractBitcoinFlinkInputFormat<BitcoinTransaction> implements CheckpointableInputFormat<FileInputSplit, Tuple2<Long,Long>> {
-	
+
 
 
 	private static final Log LOG = LogFactory.getLog(BitcoinBlockFlinkInputFormat.class.getName());
 	/**
-	 * 
+	 *
 	 */
 	private static final long serialVersionUID = 4150883073922261077L;
 	private boolean isEndReached;
 	private transient BitcoinBlock currentBitcoinBlock;
 	private long currentTransactionCounterInBlock;
-	
+
 	public BitcoinTransactionFlinkInputFormat(int maxSizeBitcoinBlock,String specificMagicStr,
-			boolean useDirectBuffer) throws HadoopCryptoLedgerConfigurationException {
+											  boolean useDirectBuffer) throws HadoopCryptoLedgerConfigurationException {
 		this(maxSizeBitcoinBlock,  specificMagicStr, useDirectBuffer,false);
 	}
-	
+
 	public BitcoinTransactionFlinkInputFormat(int maxSizeBitcoinBlock,String specificMagicStr,
-			boolean useDirectBuffer, boolean readAuxPOW) throws HadoopCryptoLedgerConfigurationException {
+											  boolean useDirectBuffer, boolean readAuxPOW) throws HadoopCryptoLedgerConfigurationException {
 		super(maxSizeBitcoinBlock,  specificMagicStr, useDirectBuffer,readAuxPOW);
 		this.isEndReached=false;
 	}
-	
```

Comment:
as the other comments, I dont think we can move it to AbstractBitcoinFlinkInputFormat. _

Reply:
Acknowledged.